cmd/spire-agent/cli/run/run.go

```diff
@@ -229,6 +233,53 @@ func mergeInput(fileInput *Config, cliInput *agentConfig) (*Config, error) {
 	return c, nil
 }
 
+func downloadTrustBundle(ac *agent.Config, c *Config) ([]*x509.Certificate, error) {
+	ac.Log.Infof("Fetching trust bundle from remote URL %s", c.Agent.TrustBundleURL)
+	resp, err := http.Get(c.Agent.TrustBundleURL)
+	if err != nil {
+		return nil, fmt.Errorf("unable to fetch trust bundle URL %s: %v", c.Agent.TrustBundleURL, err)
+	}
+
+	defer resp.Body.Close()
+	pemBytes, err := ioutil.ReadAll(resp.Body)
```

Comment:
We should check that the status code is `http.StatusOK` before bothering to read and parse the body.